src/promptflow/promptflow/_constants.py

```diff
@@ -45,6 +45,13 @@ class FlowLanguage:
     CSharp = "csharp"
 
 
+class FlowType:
+    """The enum of flow type."""
+
+    YAML_FLOW = "dag"
+    EAGER_FLOW = "flex"
```

Comment:
FLEX_FLOW?

Reply:
Is this just for flow or the thing that's being invoked in batch run?

I think we might have command, chat_group when we support more things in run.

Reply:
Regarding this name, I prefer to use YAML_FLOW and EAGER_FLOW because in other parts of our code, we use eager_flow and yaml_flow.
If we change it to dag_flow and flex_flow, it can easily cause confusion and look like a new concept different from eager and yaml.

But we can recording as dag and flex in our Telemetry first.